site/docs/faq.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: FAQ
+---
+
+## Is Skydoc used to generate the documentation on [Bazel.io](http://bazel.io)?
+
+The [Build Encyclopedia](http://bazel.io/docs/be/overview.html) documents the
```

Comment:
Add "Not for now,"

And also that we want to provide a comprehensive set dictionnary of available Skylark rule that will use skydoc.